domain-module/review/src/main/java/com/romanticpipe/reviewcanvas/service/ReviewService.java

```diff
@@ -69,14 +71,18 @@ public Review validByIdAndUserId(Long reviewId, Long id) {
 	}
 
 	public PageResponse<Review> findAllByProductId(Integer shopAdminId, Long productId, PageableRequest pageableRequest,
-												   ReviewPeriod reviewPeriod,
-												   EnumSet<ReviewFilterForShopAdmin> reviewFilters,
-												   EnumSet<Score> score, EnumSet<ReplyFilter> replyFilters) {
+		ReviewPeriod reviewPeriod,
+		EnumSet<ReviewFilterForShopAdmin> reviewFilters,
+		EnumSet<Score> score, EnumSet<ReplyFilter> replyFilters) {
 		Sort sort = SortUtils.getSort(pageableRequest.sort());
 		Pageable pageable = PageableUtils.toPageable(pageableRequest, sort);
 		return PageableUtils.toPageResponse(
 			reviewRepository.findAllByProductId(shopAdminId, productId, pageable, reviewPeriod, reviewFilters, score,
 				replyFilters)
 		);
 	}
+
+	public Integer countByShopAdminId(Integer shopAdminId) {
+		return Math.toIntExact(reviewRepository.countByShopAdminId(shopAdminId));
```

Comment:
```suggestion
		return reviewRepository.countByShopAdminId(shopAdminId).intValue();
```
위와 같이 Long을 Integer 타입으로 바꾸어도 될거 같은데 혹시 Math를 따로 사용한 이유가 있을까요?